samples/CommunityToolkit.Maui.Sample/Pages/Views/GravatarImageSourcePage.xaml

```diff
@@ -0,0 +1,129 @@
+                        <Setter Property="VisualStateManager.VisualStateGroups">
+                            <VisualStateGroupList>
+                                <VisualStateGroup x:Name="CommonStates">
+                                    <VisualState x:Name="On">
+                                        <VisualState.Setters>
+                                            <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Black}, Dark={StaticResource White}}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Off">
+                                        <VisualState.Setters>
+                                            <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Gray400}, Dark={StaticResource Gray500}}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                </VisualStateGroup>
+                            </VisualStateGroupList>
+                        </Setter>
+                    </Style>
+                    <Style TargetType="Slider">
+                        <Setter Property="MinimumTrackColor" Value="{AppThemeBinding Light={StaticResource Black}, Dark={StaticResource White}}" />
+                        <Setter Property="MaximumTrackColor" Value="{AppThemeBinding Light={StaticResource Gray200}, Dark={StaticResource Gray600}}" />
+                        <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Black}, Dark={StaticResource White}}" />
+                    </Style>
+                </ResourceDictionary>
+            </VerticalStackLayout.Resources>
+
+            <Label Style="{StaticResource Description}">
+                <Label.FormattedText>
+                    <FormattedString>
+                        <Span Text="GravatarImageSource allows you to use as an Image " />
+                        <Span Text="source" FontAttributes="Italic" />
+                        <Span Text=", a users Gravatar registered image via their email address." />
+                    </FormattedString>
+                </Label.FormattedText>
+            </Label>
+            <Line Style="{StaticResource HR}" />
+            <Image HeightRequest="128" WidthRequest="128">
+                <Image.Source>
+                    <mct:GravatarImageSource
+                        CacheValidity="{Binding CacheValidityTimespan}"
+                        CachingEnabled="{Binding EnableCache}"
+                        Email="{Binding Email}"
+                        Image="{Binding DefaultGravatarSelected}"
+                        SemanticProperties.Description="GravatarImageSource for an Image contron, binding all properties." />
```

Comment:
```suggestion
                        SemanticProperties.Description="GravatarImageSource for an Image control, binding all properties." />
```

Reply:
@bijington Despite spell checks, my dyslexia still misses things like this!